app/react/V2/Components/UI/ConfirmationModal.tsx

```diff
@@ -8,12 +8,13 @@ type confirmationModalType = {
   size?: modalSizeType;
   header?: string | React.ReactNode;
   body?: string | React.ReactNode;
-  onAcceptClick?: () => void;
+  onAcceptClick?: (value: string) => void;
```

Comment:
My thinking here is that later on we might want to return other things than a password when the user confirms. I'm going to change the internal state name so that it's `inputValue` so that it's more consistent.